wp-job-manager-functions.php

```diff
@@ -1703,3 +1708,35 @@ function job_manager_get_salary_unit_options( $include_empty = true ) {
 	 */
 	return apply_filters( 'job_manager_get_salary_unit_options', $options, $include_empty );
 }
+
+/**
+ * Checks if the job expiry can be extended.
+ * This is true if the job is public, the option to extend is set and the job expires within 5 days.
+ *
+ * @since $$next_version$$
+ *
+ * @param int|WP_Post $job The job or job ID.
+ *
+ * @return bool
+ */
+function job_manager_job_can_be_extended( $job ) {
+	$job    = get_post( $job );
+	$status = get_post_status( $job );
+	$expiry = strtotime( get_post_meta( $job->ID, '_job_expires', true ) );
```

Comment:
I think we need to call `get_job_expiration` here as it has some more logic into it.

Reply:
So we have these options:

1. `WP_Job_Manager_Post_Types::instance()->get_job_expiration( $job )->getTimestamp();`
2. `strtotime( get_post_meta( $job->ID, '_job_expires', true ) );` 

Both produce a timestamp with a day difference, I find the second one more accurate because it calculates the exact days (eg if we set 5 days to renewal, a date that is 5 days to the expiration returns true) while the `get_job_expiration` method seems a day late (It only returns true from 4 days to expiration).

Reply:
So the thing is that `get_job_expiration` uses the `job_manager_jobs_expire_end_of_day` filter which can be modified by users and return different results. For this reason, the `_job_expires` meta is only read inside `get_job_expiration` and we use this method everywhere else.